CMakeLists.txt

```diff
@@ -285,7 +290,7 @@ if(WITH_PMEM)
 endif()
 
 if(WITH_BLUESTORE)
-  if(NOT AIO_FOUND AND NOT WITH_SPDK AND NOT WITH_PMEM)
+  if(NOT AIO_FOUND AND NOT HAVE_POSIXAIO AND NOT WITH_SPDK AND NOT WITH_PMEM)
```

Comment:
@tchaikov 
I think this needs to change into:
```
 if(NOT (AIO_FOUND OR HAVE_POSIXAIO) AND NOT WITH_SPDK AND NOT WITH_PMEM)
```
So the ORed term is true if any of the 2 AIOs is available.

Reply:
no, i think it's correct and it follows the existing pattern.

Reply:
@tchaikov 
Running it again.